datanode/gateway/graphql/resolvers.go

```diff
@@ -855,6 +855,161 @@ func (r *myQueryResolver) EstimateOrder(
 	}, nil
 }
 
+func (r *myQueryResolver) EstimateFees(
+	ctx context.Context,
+	market, party string,
+	price *string,
+	size string,
+	side vega.Side,
+	timeInForce vega.Order_TimeInForce,
+	expiration *int64,
+	ty vega.Order_Type,
+) (*FeeEstimate, error) {
+	order := &types.Order{}
```

Comment:
Why do we have an order here?

Reply:
well, it's unused, lets just remove it?

Reply:
Do you mean the `order` or `err`? Order is clearly being used, I've removed the unused error from the methods.